library/init/meta/widget/interactive_expr.lean

```diff
@@ -0,0 +1,218 @@
+/- Author: E.W.Ayers -/
```

Comment:
Please follow the header style.  (It's the same as in mathlib.)

Reply:
What is the correct header to give C++ files? Should it mention microsoft?

Reply:
Exactly like in mathlib:
```c++
/*
Copyright (c) 2020 Ed Ayers. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.

Author: Ed Ayers
*/
```